package proto;

// 服务器类型，新加都需要在填写
enum ServerType {
	SVR_TYPE_ROUTER = 0x1;		// 路由服
	SVR_TYPE_CONN = 0x2;		// 客户端连接服务
}

message SocketClientIn {
}

message SocketClientOut {
}